app/temp/server.ts

```diff
@@ -1,7 +1,7 @@
-const express = require('express')
-const bodyParser = require('body-parser')
-const cors = require('cors')
-const logger = require('winston')
+const express = require('express');
```

Comment:
In the future I'd love to see the `import` syntax being used instead of `require`.

Reply:
Old file, anything in `/app/temp` is depricated and likely to die anyway.